src/content/components/BugList/columnTransforms.js

```diff
@@ -146,8 +147,16 @@ export const columnTransforms = {
     return formatDate(value, true);
   },
   priority(priority) {
+    function openPriorityGuide() {
+      prefs.set("priority_guide_collapsed", false);
+    }
+
     return (
-      <span className={priorityStyles[priority.toLowerCase()]}>{priority}</span>
+      <button
+        className={priorityStyles[priority.toLowerCase()]}
+        onClick={priority == "--" ? null : openPriorityGuide.bind(this)}>
```

Comment:
No need for `bind` since it doesn't use `this`

The reason you would not use an arrow function in the prop value is to avoid re-creating the function on every render. But this doesn't avoid that, since the function statement is inside the render method, and isn't protected behind a `useCallback` hook or anything. This isn't a functional React component, so the hook isn't an option...

So basically, `function openPriorityGuide` belongs outside of columnTransforms. It should just be defined up, outside of this block.

Btw, I actually don't like `priority_guide_collapsed` - it seems inverted. Can you change it to `priority_guide_open` and just flip its default value to true?